src/AutoBuild.jl

```diff
@@ -2,7 +2,7 @@ export build_tarballs, autobuild, print_artifacts_toml, build, get_meta_json
 import GitHub: gh_get_json, DEFAULT_API
 import SHA: sha256, sha1
 using Pkg.TOML, Dates, UUIDs
-using RegistryTools, Registrator
+using RegistryTools
```

Comment:
@DilumAluthge can you please confirm `Registrator` doesn't export any symbol?  If not, I don't think we use it here anywhere.

Reply:
Registrator doesn't export anything. But you do use a Registrator function in src/wizard/GitHub.jl.

Reply:
I know, but that's a different module :slightly_smiling_face: 
https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/fe019a6f4e1c5a218be0be7dce8df6601d2e3810/src/wizard/github.jl#L1